src/site/content/en/handbook/content-checklist/index.md

```diff
@@ -10,9 +10,9 @@ description: |
 This is a checklist for web.dev content reviews. It includes critical high-level issues and lower-level issues that commonly appear in drafts.
 
 ## Basics
-1. Is the piece providing value for one or more of the web.dev [personas](/handbook/audience)?
```

Comment:
does "piece" mean blog posts/articles/docs? This feels kind of strange and not commonly used. I'm also not sure how it would translate when you're thinking about localization.

Reply:
Yes, need an umbrella term that includes both posts and codelabs. Wasn't thrilled with _piece_ either. Suggestions welcome.

Reply:
I replaced `piece` with `content`